tcms/testcases/tests/test_views.py

```diff
@@ -564,3 +566,31 @@ def test_disabled_priority_now_shown(self):
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertContains(response, 'Set P3')
         self.assertNotContains(response, 'Set P4')
+
+
+def _call_get_selected_testcases(pks):
+    return get_selected_testcases(RequestFactory().post(reverse('testcases-clone'), {'case': pks}))
```

Comment:
Again, you don't need a helper function which directly call another function. Use `get_selected_testcases()` directly!